src/aatoolbox/datasources/ecmwf/realtime_seas5_monthly.py

```diff
@@ -173,7 +175,18 @@ def get_processed_path(self):
     @staticmethod
     def _preprocess(ds_date: xr.Dataset):
         """Set coordinate types and remove irrelevant dimensions."""
-        ds_date = ds_date.rename({"forecastMonth": "step"}).assign_coords(
+        ds_date = ds_date.rename(
+            {
+                "forecastMonth": "step",
+                # arghh because we only have one time step
+                # till now, this doesn't work nicely since
+                # it is then only dependent on step instead
+                # of time and step. This then gives errors
+                # when trying to merge with the api data
+                # TODO: help I don't know how to solve this..
+                # "verifying_time":"start_forecast_time"
```

Comment:
If there is a merging issue you may need to do add this coordinate to `expand_dims`